redis/tests/support/mod.rs

```diff
@@ -14,7 +14,7 @@ pub fn current_thread_runtime() -> tokio::runtime::Runtime {
     let mut builder = tokio::runtime::Builder::new_current_thread();
 
     #[cfg(feature = "aio")]
-    builder.enable_io();
+    builder.enable_all();
```

Comment:
Is there a reason to change this? It's effectively the same as what we already have, isn't it?

Reply:
I think that `enable_time` wasn't there when I originally wrote this. reverting.